tests/test_shell.py

```diff
@@ -280,6 +280,14 @@ def test_find(self):
                 == [job.id for job in project.find_jobs({"doc.b": i})][0]
             )
 
+        # ensure that there are no errors due to adding sp and doc prefixes
+        # by testing on all the example complex expressions
+        from test_find_command_line_interface import FILTERS
```

Comment:
Can we define `FILTERS` in some common file, maybe in `conftest.py` as a fixture?

Alternatively, can we move all `import` statements to the top of the file?

Reply:
I moved the import up, but I like having FILTERS defined near where they are most used.

Reply:
Test files should not import from other test files. It is generally a good design principle for test files to be independent of one another.

Reply:
Can we refactor that before merging?